src/python/pants/testutil/pants_integration_test.py

```diff
@@ -125,13 +125,20 @@ def run_pants_with_workdir_without_waiting(
     use_pantsd: bool = True,
     config: Optional[Mapping] = None,
     extra_env: Optional[Mapping[str, str]] = None,
-    print_exception_stacktrace: bool = True,
+    print_stacktrace: bool = True,
     **kwargs: Any,
 ) -> PantsJoinHandle:
+    if "print_exception_stacktrace" in kwargs:
```

Comment:
Is this supposed to raise a warning to the user when they run integration tests in their own pants plugin?

Reply:
Yes